CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.1.0]
```

Comment:
Hmm... Should this be a patch update instead of a minor update since there aren't any new capabilities?

Reply:
It seems like a breaking change to me. This changes the middleware options, which reference the block tracker directly.

Reply:
Oh? I may have misunderstood your description here: https://github.com/MetaMask/eth-json-rpc-middleware/pull/188#issue-1569752063 ?

Reply:
Ah, sorry, what I meant was that the way `eth-block-tracker` is used within `eth-json-rpc-middleware` was not affected by the breaking changes. I didn't mean users of this package were not affected.

Though.... looking at this more closely, maybe this is non-breaking :thinking: None of the breaking changes between eth-block-tracker v5 and v6 seem to impact this. If the only relevant change is the callback error type being changed from Error to unknown, then this should be fine as a patch.

Reply:
That said... I think both the old and new `sendAsync` types referenced here are wrong :facepalm:. The type seems to indicate that a response is mandatory. But we don't have a response in the error case.

Reply:
> That said... I think both the old and new sendAsync types referenced here are wrong 🤦. The type seems to indicate that a response is mandatory. But we don't have a response in the error case.

Are you talking about this: https://github.com/MetaMask/eth-json-rpc-middleware/blob/main/src/types.ts#L29 ?